Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -118,9 +118,29 @@
     color : "color() * acos(degrees(${temperature})) + color() * asin(${temperature}) + color() * atan(${temperature})"
 });
 
+addStyle('ATan2', {
+    color : "color() * atan2(${POSITION}[0],${temperature}) + color() * atan2(${POSITION}[1],${temperature}) + color() * atan2(${POSITION}[2],${temperature})",
+    pointSize : "5"
+});
```

Comment:
With https://github.com/AnalyticalGraphicsInc/cesium/pull/4628 in just incorporate atan2 to the Trigonometric Functions style.